### ISSUE TITLE
2.5.0 - Notes Tab Is Now Missing?

### ISSUE THREAD

**Issue by 0pLuS0** [keepassxreboot/keepassxc#3723]
2.5.0 - Notes Tab Missing?

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
Notes Tab appeared in version 2.4.3, now it's gone in 2.5.0

## Current Behavior
[NOTE]: # ( Tell us what actually happens )
When clicking on an entry, I now see these Tabs;

```General - Advanced - Autotype```

## Possible Solution
[NOTE]: # ( Not required, but suggest a fix/reason for the bug )
This appears to be a BUG?

## Steps to Reproduce
[NOTE]: # ( Provide a link to a live example, or an unambiguous set of steps to )
[NOTE]: # ( reproduce this bug. Include code to reproduce, if relevant )
1. I compiled 2.5.0 in Linux
2. When I started 2.5.0 for the first time I noticed this

These are my COMPILE FLAGS I used;

export LDFLAGS="-ltermcap"

mkdir -p build
cd build
  cmake \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_BUILD_TYPE=Release \
    -DWITH_TESTS=OFF \
    -DWITH_XC_BROWSER=OFF \
    -DWITH_XC_YUBIKEY=OFF \
    -DWITH_XC_NETWORKING=OFF \
    -DWITH_XC_AUTOTYPE=ON \
    -DWITH_XC_SSHAGENT=OFF \
    -DWITH_XC_KEESHARE=OFF \
    -DWITH_XC_KEESHARE_SECURE=OFF \
    -DWITH_XC_UPDATECHECK=OFF \
    ..

  make
  make install DESTDIR=$PKG

## Context
[NOTE]: # ( How has this issue affected you? What unique circumstances do you have? )
This was a nice option to easily access the Notes in 2.4.3 that I no longer have for some reason...

## Debug Info
[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.5.0

Libraries:
- LIBS
gcc-5.5.0 - qt5-5.9.8 - cmake-3.14.4 - qrencode-4.0.2 - argon2-20190702 - libsodium-1.0.18

Operating system: Slackware 14.2 
CPU architecture: x86_64
Kernel: 4.19.78

```Please see the attached screen shot which now shows no Notes Tab```

![notes-tab](https://user-images.githubusercontent.com/36190223/67820317-ae7b1200-fab0-11e9-957a-bf223581740a.jpg)

**Comment by MyXelf** [comment]
According to the changelog of 2.5.0, the #3336 states that the Notes field is now part of the General tab.

**Comment by 0pLuS0** [comment]
Ahh ok, my bad, let me look this back over and see... 

Also before in 2.4.3 it use to show these two TABS too;

```Attributes - Attachments```

Have these also been merged/changed?

I've attached a screen shot showing the Tabs in 2.4.3.

![2 4 3-tabs](https://user-images.githubusercontent.com/36190223/67820755-74127480-fab2-11e9-90af-949b1502b8ca.jpg)


THANKS

**Comment by droidmonkey** [comment]
They are now under advanced

**Comment by 0pLuS0** [comment]
Sorry my bad, wasn't paying attention... :(

THANKS

**Comment by 0pLuS0** [comment]
@droidmonkey

I now see a problem with this new feature of having Notes listed under General.

If I click to edit the entry under the Notes I have 2 Lines of information which is what I need, now with the notes listed under the General Tab, everything is being run together as 1 line which creates a problem of trying to distinguish the information now. 

Hmm maybe the old way with the ``` Notes Tab``` was better....

**Comment by mlippert** [comment]
@droidmonkey I see you labeled this a duplicate, which issue is it a duplicate of?

While I can certainly edit the entry to see my notes formatted as I entered them, I agree with @0pLuS0 that it might be better to show the notes as they are entered (ie keep line breaks) and have a vertical scrollbar if needed. Or show the 1st 2 or 3 lines of the notes on the General tab as a summary (perhaps w/ a ... if there are more lines) and also have the separate notes tab to show the entire notes field.

Just a thought. If I hadn't seen this issue it would have been a while before I noticed this change because I don't have to reference the notes that often.
